docs/competitive-programming/prefix-sum-algorithm.mdx

```diff
@@ -0,0 +1,179 @@
+---
+title: Prefix Sum Algorithm
+slug: /competitive-programming/prefix-sum-algorithm
+---
+
+## Introduction
+
```

Comment:
I think the Intro can be paraphrased to make it more engaging. Something like, In this article we will do ___